transformer/file2ByteArray.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable max-len */
+import {Replacer} from './index.js';
+import * as fs from 'fs';
+
+/**
+ * File2ByteArray
+ *
+ * Replace the fileToByteArray call by a Static Array<u8> including the file 
+ * assemblyscript code that use unittest functions.
+ */
+class File2ByteArray extends Replacer {
+  /**
+     * Filters all standard (library) files.
+     *
+     * @param {Source} src
+     * @return {bool}
+     */
+  isToTransform(src) {
+    return !src.isLibrary && !src.internalPath.startsWith(`~lib/`);
+  }
+
+  /**
+   * Replaces the fileToByteArray Call
+   *
+   * 
+   *
+   * @param {Node} node
+   * @return {Node}
+   */
+  visitCallExpression(node) {
+    if (node.expression.text == 'fileToByteArray') {      
+
+      const data = JSON.stringify(fs.readFileSync(node.args[0].value).toJSON().data);  
+      const result = `const bytes : StaticArray<u8> = ${data} `;
```

Comment:
here the transformer is creating an as variable "bytes". i think its pretty weird and it will cause some linter errors.
can you keep `const bytes : StaticArray<u8> = ` in the deployer function an just set the ${data} with the transformer?

Reply:
indeed !